packages/caip/src/caip10/caip10.ts

```diff
@@ -2,6 +2,8 @@ import { CAIP2, ChainNamespace, isCAIP2 } from './../caip2/caip2'
 
 export type CAIP10 = string
 
+export type AccountId = string
```

Comment:
I'm not sure that adding this additional type alias bring any value, honestly.  But it doesn't harm anything.

Reply:
@cjthompson it adds a huge amount of value.

replace all these with strings and tell me what's going on https://github.com/shapeshift/web/blob/develop/src/state/slices/portfolioSlice/portfolioSliceCommon.ts#L39-L89